app/lib/dttp/code_sets/statuses.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Dttp
+  module CodeSets
+    module Statuses
+      MAPPING = {
+        "Awaiting QTS" => { entity_id: "1b5af972-9e1b-e711-80c7-0050568902d3" },
+        "Awarded EYTS" => { entity_id: "2d5af972-9e1b-e711-80c7-0050568902d3" },
+        "Awarded QTS" => { entity_id: "195af972-9e1b-e711-80c7-0050568902d3" },
+        "Draft record" => { entity_id: "2b5af972-9e1b-e711-80c7-0050568902d3" },
+        "EYTS Revoked" => { entity_id: "2f5af972-9e1b-e711-80c7-0050568902d3" },
+        "Left course before the end" => { entity_id: "235af972-9e1b-e711-80c7-0050568902d3" },
+        "Prospective trainee - TREFNO requested" => { entity_id: "275af972-9e1b-e711-80c7-0050568902d3" },
+        "QTS revoked" => { entity_id: "155af972-9e1b-e711-80c7-0050568902d3" },
+        "Standards met" => { entity_id: "1f5af972-9e1b-e711-80c7-0050568902d3" },
+        "Standards not met" => { entity_id: "215af972-9e1b-e711-80c7-0050568902d3" },
+        "Trainee deferred" => { entity_id: "1d5af972-9e1b-e711-80c7-0050568902d3" },
```

Comment:
This PR only uses this "Trainee deferred" DTTP status, however I've included all the statuses we know about for completeness.

Reply:
Check against the list in this card: https://trello.com/c/Dl2iaFfC/867-spike-what-field-value-do-we-set-for-withdraw-and-defer-on-dttp